python/running.md

```diff
@@ -1,13 +1,9 @@
 # Running Python
 
 To start using Python, we need access to the `python` program in a terminal.
-The version installed on lxplus is 2.6.6, which is very old. However, we can get
-a newer version along with various useful packages
-(see details on the [LCG stacks][lcg_stack]) by setting up an LCG environment:
-
-```bash
-$ source /cvmfs/sft.cern.ch/lcg/views/setupViews.sh LCG_94python3 x86_64-slc6-gcc62-opt
-```
+The version installed on lxplus is 3.8.6 which can be used in these lessons.
```

Comment:
This Python isn't installed on the lxplus nodes themselves, but is made available to LHCb users via the `LbEnv` environment that's source upon login. Can clarify this to:

```suggestion
The version that's automatically made available on lxplus is 3.8.6, which can be used in these lessons.
```

@chrisburr do you reckon it's worth going in to the details of where Python comes from? I'm not sure it is, but if so we could add a footnote or something.

Reply:
ping @chrisburr

Reply:
This is a little messy as this is under `hsf-training` but only LHCb members will have Python 3.8.6 on lxplus. Perhaps this should link out to a new page on `starterkit-lessons` that explains the various environments, i.e. an expanded version of:

* When you login: Provides the basic commands for working in lhcb. Shouldn't be used for analysis.
* `lb-run`: Used for opening a shell inside a specific version of a physics application
* `lb-dirac`: Used for getting the LHCbDIRAC environment
* `lb-conda`: Used for getting a stable environment for other use cases, most commonly for doing analysis on ntuples outside of the LHCb physics applications 

Thoughts?

Reply:
I'd forgotten about non-LHCb audiences. Your suggestion sounds good to me :+1:

Reply:
Sounds good this can be edited to point to a new page for LHCb. Now maybe we can still leave the LCG setup so it can be valid for everyone at CERN or we expect someone else to contribute from other experiments for their envoronments?